client/src/components/Checkout/Checkout.js

```diff
@@ -1,264 +1,196 @@
+                    <div className="col-md-4">
+                        <div className="card">
+                            <div className="card-header">
+                                <h4>Payment method</h4>
+                            </div>
+                            <div className="card-body">
+                                <div className="row">
+                                    <div className="col-md-12">
+                                        <div className="info-group mb-3">
+                                            <label>Card holder</label>
+                                            <input
+                                                type="text"
+                                                className="info-input"
+                                                name="firstname"
+                                            />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-12">
+                                        <div className="info-group mb-3">
+                                            <label>Card number</label>
+                                            <input
+                                                type="text"
+                                                className="info-input"
+                                                name="firstname"
+                                            />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-6">
+                                        <div className="info-group mb-3">
+                                            <label>Expiry date</label>
+                                            <input
+                                                type="text"
+                                                className="info-input"
+                                                name="firstname"
+                                            />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-6">
+                                        <div className="info-group mb-3">
+                                            <label>CVV</label>
+                                            <input
+                                                type="text"
+                                                className="info-input"
+                                                name="firstname"
+                                            />
+                                        </div>
+                                    </div>
+                                    <div className="col-md-12">
+                                        <div className="info-group mb-3">
+                                            <button type="button" className="btn">
```

Comment:
i don't think you need to specify that the type is button if you're using a button tag? unless this is a bootstrap thing, then ignore this comment